spotify_wrapped.py

```diff
@@ -1,128 +1,172 @@
+    print(f"MARCH SONG NUMBER: {len(df[df_date.str.contains('March')])} (ROUGHLY {3*len(df[df_date.str.contains('March')]) / 60} HOURS)")
+    print(f"APRIL SONG NUMBER: {len(df[df_date.str.contains('April')])} (ROUGHLY {3*len(df[df_date.str.contains('April')]) / 60} HOURS)")
+    print(f"MAY SONG NUMBER: {len(df[df_date.str.contains('May')])} (ROUGHLY {3*len(df[df_date.str.contains('May')]) / 60} HOURS)")
+    print(f"JUNE SONG NUMBER: {len(df[df_date.str.contains('June')])} (ROUGHLY {3*len(df[df_date.str.contains('June')]) / 60} HOURS)")
+    print(f"JULY SONG NUMBER: {len(df[df_date.str.contains('July')])} (ROUGHLY {3*len(df[df_date.str.contains('July')]) / 60} HOURS)")
+    print(f"AUGUST SONG NUMBER: {len(df[df_date.str.contains('August')])} (ROUGHLY {3*len(df[df_date.str.contains('August')]) / 60} HOURS)")
+    print(f"SEPTEMBER SONG NUMBER: {len(df[df_date.str.contains('September')])} (ROUGHLY {3*len(df[df_date.str.contains('September')]) / 60} HOURS)")
+    print(f"OCTOBER SONG NUMBER: {len(df[df_date.str.contains('October')])} (ROUGHLY {3*len(df[df_date.str.contains('October')]) / 60} HOURS)")
+    print(f"NOVEMBER SONG NUMBER: {len(df[df_date.str.contains('November')])} (ROUGHLY {3*len(df[df_date.str.contains('November')]) / 60} HOURS)")
+    print(f"DECEMBER SONG NUMBER: {len(df[df_date.str.contains('December')])} (ROUGHLY {3*len(df[df_date.str.contains('December')]) / 60} HOURS)")
+
+print("\n")
+
+counts_1 = Counter(wrapped[wrapped_artist])
+counts_2 = Counter(wrapped[wrapped_song])
+
+most_popular_artist = dict()
+most_popular_song = dict()
+
+print(f"I LISTENED TO {len(counts_1.items())} DIFFERENT ARTISTS IN 2024\n")
+
+print(f"I LISTENED TO {len(wrapped)} SONGS IN 2024 (ROUGHLY {3*len(wrapped)} MINUTES OR {3*len(wrapped) / 60} HOURS OR {3*len(wrapped) / 60 / 60} DAYS) \n")
+
+print(f"I LISTENED TO {len(counts_2.items())} DIFFERENT SONGS IN 2024\n")
+
+print("_________________________________________________________\n")
+
+for key, value in counts_1.items():
+    if value >= 2: # Looks at how many artists you've listened to more than ten times
+        most_popular_artist[key] = value
+
+for key, value in counts_2.items():
+    if value >= 2: # Looks at how many songs you've listened to more than fifteen times
+        most_popular_song[key] = value
+
+
+most_popular_artist = (dict(sorted(most_popular_artist.items(), key=lambda x:x[1], reverse = True)))
+most_popular_song = (dict(sorted(most_popular_song.items(), key=lambda x:x[1], reverse = True)))
+
+keys_list_artist = list(most_popular_artist.keys())
+values_list_artist = list(most_popular_artist.values())
+
+#print(f"ARTISTS WITH MORE THAN 10 PLAYS IN {current_month}:\n")
+
+print("MY TOP TEN ARTISTS ON SPOTIFY OF 2024")
+
+try:
+    for i in range(0, 10): #range(len(keys_list_artist)): # Provides your top ten artists, if you want all artists more >= 10, change range to commented
+        print(values_list_artist[i], keys_list_artist[i])
+except IndexError:

```

Comment:
If you provide a number out of bounds, this error will occur. Catch it to tell the user what went wrong.